py_linq/py_linq.py

```diff
@@ -607,6 +607,18 @@ def take_while(self, predicate):
         """
         return TakeWhileEnumerable(Enumerable(iter(self)), predicate)
 
+    def to_dict(self, key=lambda x: x, value=lambda x: x):
```

Comment:
Apologies for the pickiness, but I think it might be better to stick with the `System.Linq.Enumerable` API and rename this function as `todictionary` to be consistent with the .NET API itself. My goal for this library was to have a 1-1 mapping of the API so that a C# developer could easily integrate and already know what the functions do. I am open to negotiation on this if you feel strongly about it, just wanted to give you my thoughts.

Reply:
Alright. I'll change that right away.

Reply:
@viralogic Could you please make those last changes, if possible? I removed the fork, so I can't commit to this PR. If you can't edit this, I can create a new one.